dank.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
+dfxDir="/home/dan/.config/dfx"
```

Comment:
I will move this out into a separate config file, either in this or in the next PR.